packages/rpc-graphql/src/context.ts

```diff
@@ -2,13 +2,59 @@ import { SolanaRpcMethods } from '@solana/rpc-core';
 import { Rpc } from '@solana/rpc-transport/dist/types/json-rpc-types';
 
 import { createGraphQLCache, GraphQLCache } from './cache';
+import { AccountQueryArgs } from './schema/account/query';
 
 export interface RpcGraphQLContext {
     cache: GraphQLCache;
+    resolveAccount(args: AccountQueryArgs): ReturnType<typeof resolveAccount>;
     rpc: Rpc<SolanaRpcMethods>;
 }
 
+// Default to jsonParsed encoding if none is provided
+async function resolveAccount(
+    { address, encoding = 'jsonParsed', ...config }: AccountQueryArgs,
+    cache: GraphQLCache,
+    rpc: Rpc<SolanaRpcMethods>
+) {
+    const requestConfig = { encoding, ...config };
+
+    const cached = cache.get(address, requestConfig);
+    if (cached !== null) {
+        return cached;
+    }
+
+    const account = await rpc
+        .getAccountInfo(address, requestConfig as Parameters<SolanaRpcMethods['getAccountInfo']>[1])
+        .send()
+        .then(res => res.value);
```

Comment:
Does `graphql` do something sensible if this promise throws?

Reply:
I'm not really sure, but I can explicitly bubble the error up